app/posts/publish-teacher-training-courses/2024-06-21-rollover-2024-changes.md

```diff
@@ -0,0 +1,102 @@
+
+We also merged the two fields/pages Fee Details and Financial Support because data showed that providers tended to enter all financial information into fee details
+
+Additionaly, we moved the Markdown Guidance from the right-hand column to be in-situ above the text area so it's easier to refer too
+
+![Screenshot of course fees page](course-fees.png "Course fees page")
+![Screenshot of course length](course-length.png "Course length page")
+![Screenshot of financial support](financial-support.png "Financial support page")
+
+### 2. Adding in Markdown Guidance
+
+Some pages with text areas were missing the Markdown guidance, we added these in to the following pages:
+
+- About the Accredited Provider
+- Training with your organisation
+- Training with disabilities and other needs
+
+![Screenshot of accredited providers](accredited-providers.png "Accredited providers page")
+![Screenshot of about organisation](about-org.png "About organisation page")
+
+### 3. Removing Personal Qualities and Other Requirements
+
+Whilst originally we were looking to break these sections down into seperate pages, further analysis showed that the content in the personal qualities and other requirements fields were commonly repeated across all courses - making them less useful to candidates who are looking for things that make courses different from one another.
+
+We already surface information which is very similar to the content that providers are entering in personal qualities on the screen where candidates enter their personal statement. We feel that this is the most relevant place for this content in the journey.
+
+We aren’t dropping columns because we’ll need to see the data for providers to see for courses in the past.
+
+It is going to sit behind a feature flag
+
+### 4. Changes to Course Details
+
+As we had made some changes to title changes on other pages - this information on course details page was also updated.
+
+There had also been some work done on the prototype which never went live, including:
+
+- Removing the truncation so that all copy is visible
+- Using the 'Tab' design for the Basic Details / Description links
+- Removal of Personal Qualities
+- Removal of Other Qualities
+
+![Screenshot of courses](courses.png "Courses page")
+
+### 5. SEND checkbox to radio buttons
+
+To match how the SEND option appears in the edit section, we changed the SEND question from a checkbox to radio buttons
+
+![Screenshot of send option](send.png "SEND option")
+
+### 6. Full Time or Part Time radio buttons to checkboxes
+
+This has been changed from radio buttons to checkboxes to follow best practises. Users can now chose either or both.
+
+![Screenshot of full time or part time option](full-part-time.png "Full time or part time option")
+
+### 7. Add a Second Language validation
+
+Some validation has been added so that users cannot select the same language as a secondary option that they already selected as a primary option
```

Comment:
I wonder if primary and secondary are confusing here only because of the context of education! Maybe in this case just use first and second (even though primary and secondary are correct)?